.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Which version do you use? Provide branch and commit id.
+2. Which build settings do you use? 
+3. On which machine does your problem occur?
+4. Provide parameter/material files.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
```

Comment:
Also: actual behavior?

Reply:
Actual behaviour should be given in "describe the bug"
But I'll add more text to make it unambiguous.